.github/SECURITY.md

```diff
@@ -0,0 +1,22 @@
+# Security Policy
+
+## Supported Versions
+
+| Version       | Supported          |
+| ------------- | ------------------ |
+| 1.1.1         | :white_check_mark: |
+| < 1.1.1       | :x:                |
+
+## How we do security
+
+
+As much as possible, AutoGluon relies on automated tools to do security scanning. In particular, we support:
+
+1. Dependency Analysis: Using Dependabot
+2. Docker Scanning: Using Snyk
+3. Code Analysis: Using CodeGuru
+
+
+## Vulnerability Reporting
```

Comment:
nit: "Reporting a Vulnerability" reads slightly nicer imo. (that is the wording used by scikit-learn)

Reply:
updated